### PR TITLE
docs(widget): widget colors palette

### PR DESCRIPTION
**Pull request by shoom3301**
# Description

Updated widget docs regarding `CowSwapWidgetPalette`

### CODE REVIEW

**Comment by elena-zh** [comment]
The build has failed, can't review :(

**Review by alfetopito — APPROVED** [review]

File: docs/cow-protocol/tutorials/widget.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ import {cowSwapWidget, CowSwapWidgetParams} from '@cowprotocol/widget-lib'
 const widgetContainer = document.getElementById('cowswap-widget')
 
 const params: CowSwapWidgetParams = {
-  appKey: 'YOUR_APP_ID',
+  appCode: 'YOUR_APP_ID',
```
Comment (anxolin) [inline docs/cow-protocol/tutorials/widget.md]: nit:
```suggestion
  appCode: 'NAME-OF-YOU-APP', // Add here the name of your app. e.g. "Pig Swap"
```

File: docs/cow-protocol/tutorials/widget.md [hunk 2]
```diff
@@ -110,23 +110,60 @@ cowSwapWidget(
...
+|-----------------------|------------------------|---------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `width`               | `string`               | 400px                     | The width of the widget in css values (px, vh, etc.).                                                                                                                                                            |
+| `height`              | `string`               | 600px                     | The height of the widget in css values (px, vh, etc.).                                                                                                                                                           |
+| `appCode`              | `string`               | 'DEFAULT_INJECTED_WIDGET' | The unique identifier of the widget consumer. Please fill the for to let us know a little about you: https://cowprotocol.typeform.com/to/rONXaxHV                                                                |
```
Comment (anxolin) [inline docs/cow-protocol/tutorials/widget.md]: I think this one is mandatory

File: docs/cow-protocol/tutorials/widget.md [hunk 3]
```diff
@@ -110,23 +110,60 @@ cowSwapWidget(
...
+| `enabledTradeTypes`   | `Array<TradeType>`     | All are enabled           | CoW Swap provides three trading widgets: `swap`, `limit` and `advanced` orders. Using this option you can narrow down the list of available trading widgets.                                                     |
+| `partnerFeeBips`      | `string`               | ---                       | Coming soon! You can enable a fee for all trades in the widget. Please contact [us](https://calendly.com/crystal-cow/cow-swap-widget) to enable your partner fee.                                                |
+
+## Custom theme
+
+By setting custom colors you can very flexibly control the appearance of the widget.  
+To do this, you need to pass an object of type `CowSwapWidgetPalette` to the `theme` parameter.
+
+You actually can specify only 2 or 3 basic colors, the rest of the palette is efficiently derived (also with the help of the [color2k](https://github.com/ricokahler/color2k) library to get 'AA' rated text/background color contrasts).
```
Comment (anxolin) [inline docs/cow-protocol/tutorials/widget.md:135]: Maybe point to https://widget.cow.fi so they can experiment

File: docs/cow-protocol/tutorials/widget.md [hunk 4]
```diff
@@ -138,7 +175,7 @@ import {cowSwapWidget, CowSwapWidgetParams} from '@cowprotocol/widget-lib'
 const container = document.getElementById('cowswap-widget')
 
 const params: CowSwapWidgetParams = {
-  appKey: 'YOUR_APP_ID',
+  appCode: 'YOUR_APP_ID',
```
Comment (anxolin) [inline docs/cow-protocol/tutorials/widget.md]:
```suggestion
   appCode: 'NAME-OF-YOU-APP',
```

**Review by anxolin — APPROVED** [review]
Approved. Only nit comments

File: docs/cow-protocol/tutorials/widget.md (hunk 3 above)
Comment (shoom3301) [reply docs/cow-protocol/tutorials/widget.md:135]: There is a link bellow:

>Try it yourself: https://widget.cow.fi

**Pull request merged**